src/stores/chats.ts

```diff
@@ -583,7 +591,10 @@ export const useChatStore = defineStore('chats', {
         const message = { payloadDigest: index, ...newMsg }
         if (index in this.messages) {
           // Mutate the object so that it striggers reactivity
-          this.messages[index] = Object.assign(this.messages[index], message)
+          this.messages[index] = Object.assign(
+            this.messages[index] as Message,
```

Comment:
???

Reply:
If these coercions are necessary it should indicate there's a type problem elsewhere. Generally don't want to have to do this.